docs/overview.md

```diff
@@ -101,13 +101,13 @@ Once created, we publish this Import Map to our Eik server and then create an al
 In each application we can now depend on and install lit-html through NPM as is common practice. Each application can then locally reference lit-html through its bare import statement like so:
 
 ```js
-import * as lit from 'lit-html';
+import {html, render} from 'lit-html'
 ```
 
 In the build tool used by the applications we can now add the appropiate Eik mapping utility which will read a set of defined Import Maps (in our example, "site-mapping") from the Eik server and apply these Import Maps to the application code. This will map our bare import statements into legal ESM import statements pointing to the lit-html alias defined in the Import Map:
```

Comment:
```suggestion
In the build tool used by the applications we can now add the appropriate Eik mapping utility which will read a set of defined Import Maps (in our example, "site-mapping") from the Eik server and apply these Import Maps to the application code. This will map our bare import statements into legal ESM import statements pointing to the lit-html alias defined in the Import Map:
```